### PR TITLE
Minor lightbox style fixes

### PR DESCRIPTION
**Pull request by broskoski**

### CODE REVIEW

File: apps/block/stylesheets/index.styl [hunk 1]
```diff
@@ -63,14 +63,16 @@ block-sidebar-breakpoint = 950px
...
   margin (unit * 2) auto
-  overflow-x auto
   padding unit * 2
   text-align left
   min-height 80vh
   font-size 1.2rem
   line-height 1.7em
   max-height 75vh
-  overflow scroll
+  overflow-x scroll
```
Comment (dzucconi) [inline apps/block/stylesheets/index.styl]: why would we overflow-x?

Comment (broskoski) [reply apps/block/stylesheets/index.styl]: oops, brain

File: apps/block/stylesheets/index.styl [hunk 2]
```diff
@@ -69,10 +69,10 @@ block-sidebar-breakpoint = 950px
   font-size 1.2rem
   line-height 1.7em
   max-height 75vh
-  overflow-x scroll
+  overflow-y scroll
```
Comment (dzucconi) [inline apps/block/stylesheets/index.styl]: auto?

Comment (dzucconi) [reply apps/block/stylesheets/index.styl]: otherwise this will be always scrollbar

File: apps/block/stylesheets/index.styl [hunk 3]
```diff
@@ -69,10 +69,10 @@ block-sidebar-breakpoint = 950px
...
 
   &.is-editing
-    overflow-x visible
+    overflow-y visible
```
Comment (dzucconi) [inline apps/block/stylesheets/index.styl]: also—huh?

**Pull request merged**